tests/conftest.py

```diff
@@ -361,3 +361,17 @@ def configuration_fx() -> Dict:
     provides the session configuration dictionary, as loaded at test session start.
     """
     return ConfigReceiver.data
+
+
+@pytest.fixture
+def mock_fuelrats_api_fx():
+    # TODO pull from configuration system
+    return MockupAPI(url=r'http://api.thehellisthis.com:6543/api')
```

Comment:
[Suggestion] Would it not be better to rely on `aiohttp`'s built-in mocking system? This makes our tests depend upon an outside source, or we would have to develop and spin up a webserver during testing. See `Galaxy`'s test suite for mocking examples.

Reply:
Great suggestion, TBH I totally forgot such features existed. I may consult you and the Galaxy code to help get this up to speed 😇